tests/clients/web/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
 # All of our core javascript will be checked with a single test
-add_javascript_style_test(core src)
```

Comment:
> We should write a different JS style tool that doesn't have that bug! :wink:

Only if we can concurrently invent several brand-new style standards: "semi-unified", "unified", and "grand unified", all of which will be objectively vastly superior to existing standards. Then we'll add a [Greenkeeper](http://greenkeeper.io/)-style feature to automatically send PRs to change the style in all repos that do not use our standard.